Assets/Scripts/Core/GridCombatSystem.cs

```diff
@@ -45,6 +45,8 @@ private enum State {
 				}
 			}
 
+			UnityEngine.Debug.Log("Hit chance: " + RangedHitChance(blueTeamList_[0].GetPosition(), redTeamList_[0].GetPosition(), 100f));

```

Comment:
Why do you use `UnityEnigine.` here, if you done this correctly you should not need to use this at all.